recipes/r-data.table/1.10.4/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-data.table
+  version: "1.10.4"
+
+source:
+  fn: data.table_1.10.4.tar.gz
+  url:
+    - http://cran.r-project.org/src/contrib/data.table_1.10.4.tar.gz
+    - http://cran.r-project.org/src/contrib/Archive/data.table/data.table_1.10.4.tar.gz
+  sha256: 865fdf6aad389071ad063ec1c75a78ffc86eeb88bba011f3ea5281d243966b7a
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: bit64, knitr, chron, ggplot2 (>= 0.9.0), plyr, reshape, reshape2, testthat (>= 0.4), hexbin, fastmatch, nlme, xts, gdata, GenomicRanges, caret, curl, zoo, plm, rmarkdown, parallel
+requirements:
+  build:
+    - r
+    - posix               # [win]
+    - {{native}}toolchain # [win]
+    - gcc                 # [not win]
+
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('data.table')" # [not win]
+    - "\"%R%\" -e \"library('data.table')\"" # [win]
+
+about:
+  home: http://r-datatable.com
+  license: GPL-3 | file LICENSE
+  summary: Fast aggregation of large data (e.g. 100GB in RAM), fast ordered joins, fast add/modify/delete
+    of columns by group using no copies at all, list columns, a fast friendly file reader
+    and parallel file writer. Offers a natural and flexible syntax, for faster development.
+  license_family: GPL3
+
+# The original CRAN metadata for this package was:
+
+# Package: data.table
+# Version: 1.10.4
+# Title: Extension of `data.frame`
```

Comment:
Sure. I followed the pattern that I found in a number of other packages, such as r-dorng (I did not do a statistic though on when these were created and whether it is still fashionable to do this).

A style guide would be nice.